### ISSUE TITLE
Files crash if replace canceled

### ISSUE THREAD

**Issue by Djaler** [elementary/files#696]
Steps to reproduce:
1. Initiate replace dialog in Files
2. Click "Cancel"

Expected:
Dialog just canceled, everything else is as usual

Actual:
Files crashed

Screencast available here - https://youtu.be/qOZ0EH1SdD8

Crash message in terminal:
```
ERROR:/build/pantheon-files-JBeKj5/pantheon-files-4.0+r3132+pkg70~daily~ubuntu5.0.1/libcore/marlin-file-operations.c:4151:copy_move_file: code should not be reached
```

**Comment by arshubham** [comment]
Possible duplicate of: https://github.com/elementary/files/issues/682

**Comment by jeremypw** [comment]
The fix to #682 will fix this (the cause is the same).

**Comment by megatux** [comment]
I can reproduce it here, I was doing a drag from a directory to the left quick access list . Most probably the same issue. The error in console is the same. Is the fix on #682 still not shipped?
screencast: https://youtu.be/DZBLsaeAE7M

**Comment by jeremypw** [comment]
@megatux :  No, not yet.  The fix was merged 22 days ago, the last release (4.0) was 29 days ago.

**Comment by danrabbit** [comment]
Closing since it's a duplicate
